services/provider/server/server.go

```diff
@@ -1229,13 +1230,39 @@ func (s *OCSProviderServer) getDesiredCephConnection(
 		return nil, fmt.Errorf("failed to extract monitor IPs from configmap %s: %v", monConfigMap, err)
 	}
 
+	readAffinityOptions := util.GetReadAffinityOptions(storageCluster)
+	readaffinityDisabledAnnotation := consumer.Annotations["ocs.openshift.io/disable-read-affinity"] == "true"
```

Comment:
I would prefer if we would not hard code boolean values as strings. There are 2 alternatives:
- format an actual boolean value into the string that we care about

```suggestion
	readAffinityDisabledAnnotation := consumer.Annotations["ocs.openshift.io/disable-read-affinity"] == strconv.FormatBool(true) {
```

Or to parse the value of the annotation as a bool
```suggestion
	readAffinityDisabledAnnotation, _ := strconv.ParseBool(consumer.Annotations["ocs.openshift.io/disable-read-affinity"])
```

The second approach automatically supports multiple variation of a truth value: "true", "True", "TRUE", "1", etc.

Reply:
done